### PR TITLE
Feature/job management system

### PR DESCRIPTION
**Pull request by Ibukun-AL**
This PR implements a job management system with the following features:
- Add jobs module and related files
- Delete job listings by an admin
- Update a job post by ID
- Get job details
- Delete job details by ID

Implementation details:
- Used Spring Boot for backend implementation
- Integrated PostgreSQL for data persistence
- Implemented CRUD operations for job listings
- Added unit tests for the JobController

To test:
1. Run the application
2. Use Postman or any API client to test the endpoints:
   - GET /api/jobs
   - POST /api/jobs
   - GET /api/jobs/{id}
   - PUT /api/jobs/{id}
   - DELETE /api/jobs/{id}

Please review and let me know if any changes are needed.

### CODE REVIEW

**Comment by Am0du** [comment]
You have conflicts in your pom.xml
And don't commit your application.properties

**Comment by Am0du** [comment]
Your pom has conflicts

File: .env
Comment (lurldgbodex) [inline .env:1]: Please remove the .env file

File: src/main/java/hng_java_boilerplate/job/controller/JobController.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import hng_java_boilerplate.job.Exceptions.ResourceNotFoundException;
+import hng_java_boilerplate.job.models.Job;
+import hng_java_boilerplate.job.models.ResponseWrapper;
+import hng_java_boilerplate.job.service.JobService;
+
+@RestController
+@RequestMapping("/api/v1/jobs")
+public class JobController {
+    @Autowired
+    private JobService jobService;
```
Comment (lurldgbodex) [inline src/main/java/hng_java_boilerplate/job/controller/JobController.java:20]: constructor injection is preferred to field injection

**Review by lurldgbodex — CHANGES_REQUESTED** [review]
I can see that you create new entities but i can't find your migration file

**Comment by Fifetoyi** [comment]
Fix PR name to match convention.
Fix branch name to be `feat/...`

File: src/main/java/hng_java_boilerplate/blog/service/BlogService.java
Comment (Fifetoyi) [inline src/main/java/hng_java_boilerplate/blog/service/BlogService.java:1]: remove all comments from the entire pull request. all files should be free of comments

File: src/main/java/hng_java_boilerplate/job/Exceptions/GlobalExceptionHandler.java
Comment (Fifetoyi) [inline src/main/java/hng_java_boilerplate/job/Exceptions/GlobalExceptionHandler.java:1]: we already have a global exception handler in the codebase. if your method does not exist there then you can include it

**Pull request closed without merging**